fishcore/Controllers/FileController.cs

```diff
@@ -0,0 +1,101 @@
+﻿using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Logging;
+using Newtonsoft.Json;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+using YamlDotNet.Serialization;
+
+
+namespace fishcore.Controllers
+{
+    [Route("[controller]")]
```

Comment:
The FileController takes the development window interface info and converts it into a YAML file for later import.